index.html

```diff
@@ -0,0 +1,212 @@
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+      </style>
+      <script
+      src="https://kit.fontawesome.com/675259b560.js" crossorigin="anonymous">
+    </script>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+            <section id="home">
+                <nav id="navbar" class="open">
+                    <div class="container">
+                        <a href='#' class="logo">My Logo</a>
+                        <div class="boton"><i class="fa-solid fa-bars fa-2xs"></i></div>
+                    </div>
+                </nav>
+            </section>
+    </header>
+    <main>
+        <section class="portfolio">
+            <div>
+                <h1>Hey There. I'm Jonathan.</h1>
+            </div>
+            <div>  
+                <h2>I'm a Software Developer</h2>
+            </div>
+            <div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        dont hesitate to contact me.</p>
+            </div>
+            
+            <div class="icons">
+                <a href="">
+                    <img src="Assets/Github.svg" alt="Github">
+                </a>
+                <a href="">
+                    <img src="Assets/Linkedin.svg" alt="Linkedin">
+                </a>
+                <a href="">
+                    <img src="Assets/Icon - Angelist.svg" alt="Angelist">
+                </a>
+                <a href="">
+                    <img src="Assets/Twitter.svg" alt="Twitter">
+                </a>
+                <a href="">
+                    <img src="Assets/Medium.svg" alt="Medium">
+                </a>
+            </div>
+        </section>
+        <section id="experience">
+            
+            <h3 class="title">My Recent Works</h3>        
+            <img class="vector" src="Assets/Vector.svg" alt="">        
+            <div>
+                <img  class="placeholder" src="Assets/Img Placeholder.svg" alt="">
+                
+                <div>
+                    <h4 class="work">Multi-Post Stories</h>
+                </div>
+                <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="teclast">
+                    <div class="css-button">CSS</div>
+                    <div class="html-button">HTML</div>
+                    <div class="boot-button">Bootstrap</div>
+                    <div class="ruby-button">Ruby</div>
+                </div>
```

Comment:
- [ ] Great job on wrapping your list using **ul** tag 💚 However, you missed this part, so please remedy that detail 🙏🏽 It will improve the consistency of your code 📗